.github/workflows/pull-request.yml

```
@@ -0,0 +1,354 @@
+    permissions:
+      contents: write
+    #needs: PR-Greeting
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          # ref: ${{ github.event.pull_request.head.sha }}
+          fetch-depth: 0
+          token: ${{ secrets.GITHUB_TOKEN }}
+      - uses: actions/setup-java@v4
+        with:
+          distribution: "zulu" # See 'Supported distributions' for available options
```

YAMLlint (1.37.1)

[warning] too few spaces before comment: expected 2

(comments)